codechain/rpc/base.py

```diff
@@ -0,0 +1,37 @@
+import json
+from typing import Callable
+from typing import Union
+
+import requests
+
+
+class rpc_base:

```

Comment:
You should be consistent with the class name. We have three different naming convention on the class name. `snake_case` on `rpc_base`, and `camelCase` on `keystoreManager`, and `PascalCase` on the others. Also, we have inconsistency in file names.